taxi_service/settings.py

```diff
@@ -39,11 +38,13 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
```

Comment:
The `debug_toolbar` is included in the `INSTALLED_APPS`. Ensure this is only used in development environments, as it can expose sensitive information.